### ISSUE TITLE
turn melodies for MS 73 back on

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#1893]
Now that all melodies in this source have been proofread, we should revert  #1635 so the melodies are public and viewable!

**Comment by annamorphism** [comment]
@liampond I was surfing Cantus today and remembered this issue...any chance we can get this turned back on?

**Comment by annamorphism** [comment]
to check (from #1650):
"The following pages were changed and should be checked that they are working for MS 73 chants for this issue to be completed:

    chant detail
    global chant search
    ms chant search
    source chant browse
    source list
    melody search (global and source-specific)"

**Comment by kyrieb-ekat** [comment]
I'm working on this in a pair of PRs here: #2017 #2018

**Comment by annamorphism** [comment]
so, this seems to work (although there are a few erratically missing melodies--I'm guessing the RA missed them because they were turned off outside the edit mode!) BUT when I try to find any non-proofread chants I get a 404. is this to be expected...? https://staging.cantusdatabase.org/source/680970/chants/?page=4&volpiano_proofread=False
